scripts/update-helm-chart-reference/main.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
 HELM_CHART_DOCS_GENERATOR_VERSION=0.2.0
-DESTINATION=src/content/vintage/use-the-api/management-api/cluster-apps
+
+DESTINATION=src/content/reference/platform-api/cluster-apps
```

Comment:
We want new generated content to land in `/reference`.

We don't want the pages in `/vintage` to get updated any more.